spanner/google/cloud/spanner_v1/keyset.py

```diff
@@ -60,17 +60,19 @@ def to_pb(self):
         """
         kwargs = {}
 
-        if self.start_open:
+        if self.start_open is not None:
             kwargs['start_open'] = _make_list_value_pb(self.start_open)
-
-        if self.start_closed:
+        elif self.start_closed is not None:
             kwargs['start_closed'] = _make_list_value_pb(self.start_closed)
+        else:
+            kwargs['start_closed'] = _make_list_value_pb([])
```

Comment:
It's unclear to me why'd you prefer an **explicitly** empty value over an implicitly empty one.

Reply:
iirc, to_pb complained because I didn't supply with start keys if I only specified end_*

Reply:
Can you show me the complaint?